docs/Report.md

```diff
@@ -17,59 +17,42 @@ Provide an illustration of your domain model. Make sure that it is correct and c
+In the Onion Architecture diagram bellow you'll see our applications. In the centre we have our core package. This is the lowest layer of the application. Then we move outwards through the layers end with our  SQL-Server and razor pages, which interacts with our Azure application. 
 <br>
 
-<figure align = "center">
-    <img title="Onion Architecture Diagram" id="OnionArchitectureDiagram" style="width:30%" alt="Alt text" src="Images\OnionArchitectureDiagram.png">
-    <figcaption style="font-size:11px" ><b>Fig. XX - Onion Architecture Diagram</b></figcaption>
-</figure>
+![Onion Architecture Diagram](Images/OnionArchitectureDiagram.png){width=60%}
 
+In order not to inundate the diagram. The details of the classes are kept minimal in the Onion class diagram [OnionClassDiagram](#OnionClassDiagram). There is a UML class diagram for each package. All of these are shown in the Onion class diagram. This is done to keep the diagrams clear and readable. The interaction between layers and packages is shown in the Onion class diagram. The internal interaction is shown in the UML class Diagrams. 
 
-These layers can be seen more detailed in our class diagram. There is one for each package, and they show everything needed to know about our classes. We've chosen to do this for more simplicity in reading the diagrams. The diagrams show each package and how the classes interact with each other. To see how they interact with the other layers, see [OnionClassDiagram](#OnionClassDiagram) further down.
-
-You will see in our repositories, that we're deleting the author at some point, this was a project demand. We had two possibilities; delete the user in the sense that they will no longer be traceable, that is make everything anonymous and delete their information, or we had the possibility of just deleting everything that the user ever touched. We chose to be sure that the user wouldn't come back complaining that their username/normal name still was in a cheep, so we deleted everything that they touched. This was also the easier approach since we could delete everything that contained that user's id or name, instead of altering everything. 
+You will see in our repositories, that we're deleting the author at some point, this was a project demand. We had two possibilities; delete the user in the sense that they will no longer be traceable, that is make everything anonymous and delete their information, or we had the possibility of deleting everything that the user ever touched. We chose to be sure that the user wouldn't come back complaining that their username/normal name still was in a cheep, so we deleted everything that they touched. This was also the easier approach since we could delete everything that contained that user's id or name, instead of altering everything. 
```

Comment:
increased clarity, fixed sentence structure and adjusted wording so as not to convey the wrong meaning.

Reply:
would write overwhelmed instead of inundate in the first sentence as Helge has said he doesn't like any fancy word. It should be simple